terasoluna-gfw-functionaltest-web/pom.xml

```diff
@@ -65,6 +65,10 @@
         </dependency>
         <dependency>
             <groupId>org.terasoluna.gfw</groupId>
+            <artifactId>terasoluna-gfw-validator</artifactId>

```

Comment:
- [ ] Is this file for this PR? It looks like test implementations for validation libraries...